### PR TITLE
changed p tags to divs to account for wagtail richtext rendering

### PR DESCRIPTION
**Pull request by kave**
@jimmynotjim

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html [hunk 1]
```diff
@@ -29,9 +29,9 @@
         {
             'half_width_link_blob': {
                 'heading' :       'This is an example of a Half Width Link Blob',
-                'body':  'To highlight navigational links and provide
-                            context about what users can expect on the linked
-                            page',
+                'body': {
+                    'source' : lipsumText
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html]: Remove spaces before `:` here and throughout.

**Comment by jimmynotjim** [comment]
Small comment, otherwise :+1:

File: cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html (hunk 1 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html]: donzo

**Comment by kave** [comment]
resolves this https://github.com/cfpb/cfgov-refresh/issues/1206

**Pull request merged**